www/pages/_app.js

```diff
@@ -26,15 +26,28 @@ const App = ( props ) => {
     }
   });
 
+  useEffect( () => {
+    async function themeRefresh() {
+      await store.dispatch(
+        refreshTheme()
+      );
+    }
+
+    themeRefresh();
+  });
```

Comment:
Since we know that the `theme` will not change unless the user changes it, we can actually move this `useEffect` block to the `Layout` and add the `user` to the array of dependencies. This will keep us from making requests for the theme when we don't need to. You also don't necessarily need to `await` the dispatch since there is no logic later in the `useEffect` that relies on the outcome of the dispatch.